packages/distribution/tests/integration/components/cd-navigation/status-indicator-test.js

```diff
@@ -60,5 +60,36 @@ module(
         intl.formatDate(overdueDeadline.toJSDate()),
       );
     });
+    test("it renders the pending indicator if there are open and unanswered inquiries", async function (assert) {
```

Comment:
```suggestion
    test("it renders the pending indicator if there are unanswered inquiries", async function (assert) {
```
Open and unanswered is the same

Reply:
Done.